src/Timers.cpp

```diff
@@ -319,6 +319,23 @@ bool Timers::UpdatePvrTimer(tinyxml2::XMLNode* pRecordingNode, kodi::addon::PVRT
     }
   }
 
+  if (status == "Pending")
+  {
+    std::string directory;
+    if (XMLUtils::GetString(pRecordingNode, "directory", directory))
+    {
+        int i = 0;
+        for (auto it = m_settings.m_recordingDirectories.begin(); it != m_settings.m_recordingDirectories.end(); ++it, i++)
```

Comment:
As a question of understanding, is there a background to using the iterator and an additional "i" here and on other places?

Could here the following not enough?:
```cpp
  for (size_t i = 0; i < m_settings.m_recordingDirectories.size(); ++i)
```

Reply:
I took existing code https://github.com/kodi-pvr/pvr.nextpvr/blob/Matrix/src/Timers.cpp#L195b for a similar recording group lookup and reused it for this loop.   It dates back to Jarvis, I will change both for readability.

Reply:
@AlwinEsch I made the changes but used unsigned int rather then size_t to avoid compiler warning.

Reply:
No problem, thought only about the size() call, but you are right, the size_t becomes then more in conflict on some other places where then bring warnings.